electron/src/js/about.ts

```diff
@@ -39,11 +40,13 @@ const ABOUT_WINDOW_WHITELIST = [
   fileUrl(path.join(APP_PATH, 'img', 'wire.256.png')),
   fileUrl(path.join(APP_PATH, 'css', 'about.css')),
 ];
-const PRELOAD_JS = path.join(APP_PATH, 'js', 'preload-about.js');
+const PRELOAD_JS = path.join(APP_PATH, 'dist', 'js', 'preload-about.js');
 
-ipcMain.once(EVENT_TYPE.UI.WEBAPP_VERSION, (event, version) => (webappVersion = version));
+ipcMain.once(EVENT_TYPE.UI.WEBAPP_VERSION, (event: IpcMessageEvent, version: string) => (webappVersion = version));
 
 const showWindow = () => {
+  let aboutWindow: BrowserWindow | undefined;
```

Comment:
Let's see if we can turn this into a `const` and get rid of all the safeguards. I suppose there must be a way to destroy a `BrowserWindow`. The only reason we currently have this is that we set it to undefined later on...

Reply:
Sure, we could, but the docs [recommend to remove the reference](https://github.com/electron/electron/blob/master/docs/api/browser-window.md#event-closed) to the `BrowserWindow` object. It also helps the garbage collector to clean up after the window is closed.

Reply:
As discussed f2f, will leave it like it is.